### PR TITLE
Selected image class

### LINKED ISSUE #913: [Select]:  Differentiate selected item image class
Differentiate the selected item image class from the dropdown items image class.

### PR DESCRIPTION
**Pull request by MateusStanki**
## Closes Issues

* Closes #913

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

File: packages/input-select/src/components/withImage.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default Input =>
         <LabelContainer className="labelContainer">
           {image && (
             <ItemImage
-              className="itemImage"
+              className="selectedItemImage"
```
Comment (alcferreira) [inline packages/input-select/src/components/withImage.js]: just changed the classname here... Is it enough?

Comment (MateusStanki) [reply packages/input-select/src/components/withImage.js]: That's the pr purpose.

Comment (alcferreira) [reply packages/input-select/src/components/withImage.js]: Still didn't get based on the issue description. Can you elaborate more?

Comment (MateusStanki) [reply packages/input-select/src/components/withImage.js]: Yes, for sure. In select inputs we have the dropdown items which are the options, and the item that was selected (if there is one). We have few different styles in both, mainly using the small size. The problem is that in both cases we use the same class, which is not used internally, was created to have access to the element style at implementation time. Applying styles in this class gave different behaviors between the selected item and the others. So, doesn't make any sense to use the same class on both, correct? That's why i created this simple PR.

Comment (alcferreira) [reply packages/input-select/src/components/withImage.js]: Ohhhh now i see, i thought it was about the selected image inside the dropdown items
Thanks!

**Pull request merged**